xapian-core/tests/api_db.cc

```diff
@@ -1687,14 +1687,54 @@ DEFINE_TESTCASE(sortrel1, backend) {
     return true;
 }
 
-// Test network stats and local stats give the same results.
-DEFINE_TESTCASE(netstats1, remote) {
-    XFAIL_FOR_BACKEND("multi_glass_remoteprog_glass",
-		      "Multi remote databases are currently buggy");
+static void
+make_netstats1_db(Xapian::WritableDatabase& db, const string&)
+{
+    struct { Xapian::docid did; const string text; } content[] = {
+	{1, "This is a test document used with the API test. "
+	    "This paragraph must be at least three lines (including the blank "
+	    "line) to be counted as a \"paragraph\"."},
+	{2, "This is a second simple data test, used to test multiple "
+	    "(inmemory anyway) databases.  The text in this file is"
+	    " unimportant, although I suppose it ought to include "
```

Comment:
It would be better to be consistent about whether the space goes before or after the line break when splitting a string.  Xapian convention is to put it before, and you've mostly done that, but not here and also not in a few other cases below.

Being inconsistent about this makes it much harder for the reader to easily see that there are no missing spaces where
the string has been broken up.